src/PrinterLog.php

```diff
@@ -108,14 +109,20 @@ public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $
      *
      * @return array
      */
-    public function getMetrics(Printer $printer, $user_filters = []): array
+    public function getMetrics(Printer $printer, $user_filters = [], $interval = 'P1Y', $start_date = null, $end_date = new DateTime(), $format = 'dynamic'): array
```

Comment:
The line is a bit long and the arguments could be split into a few lines (see https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments).

Also PHPDoc is missing for the arguments